.github/workflows/trigger-message.yml

```diff
@@ -9,23 +9,8 @@ jobs:
     if: ${{ github.event.issue.pull_request && startsWith(github.event.comment.body, '/generate') }}
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout the correct pull request branch
+      - name: Checkout code
```

Comment:
what is this changes?

Reply:
when i try to push i have a conflict:To github.com:vlad-vinskevitch/app-afesox.git
 ! [rejected]        feature/SITIONIX-4/5 -> feature/SITIONIX-4/5 (fetch first)
error: failed to push some refs to 'github.com:vlad-vinskevitch/app-afesox.git'
hint: Updates were rejected because the remote contains work that you do
hint: not have locally. This is usually caused by another repository pushing
hint: to the same ref. You may want to first integrate the remote changes
hint: (e.g., 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
 need to merge with branch